index.html

```diff
@@ -22,6 +22,7 @@
     <main>
       <h1 class="hidden">Добро пожаловать в Джорждию!</h1>
       <section  class='advertisement'>
+        <h2>Рекламный блок</h2>
```

Comment:
Указал заголовок для секции (нужно будет скрыть) 
```suggestion
        <h2>Рекламный блок</h2>
```